src/assets/icons/burger-2.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd">
+<defs>
+    <style>
+      .cls-1 {
+        fill: rgb(218, 165, 32));
```

Comment:
I'd stick to one approach with coloring icons - take a look here: https://github.com/akai-org/polifood-client/pull/24/files